### ISSUE TITLE
Apache's directory indexes could include hashes - or a link to them

### ISSUE THREAD

**Issue by poeml** [poeml/mirrorbrain#43]
[          ]
                                                                                     
#### Issue migrated (2015-06-05) from old issue tracker http://mirrorbrain.org/issues/issue43

```
Title    Apache's directory indexes could include hashes - or a link to them
 Priority   wish                     Status            resolved
Superseder                          Nosy List          poeml
Assigned To poeml                   Keywords

```

##### msg141 (view)        Author: poeml            Date: 2010-03-08.21:20:13
(blocked by issue 40)

The directory indexes that Apache generates could include links to hashes, like
md5 and sha1. In fact, they could even display the hashes directly, if that would
make sense.

##### msg143 (view)        Author: poeml            Date: 2010-03-08.21:23:01
From the old TODO file: 'add "md5" and "sha1" or "checksums" link to
mod_autoindex_mb. Make it configurable so it can be switched off for the very
large directories that are already slow to show anyway.'

##### msg225 (view)        Author: poeml            Date: 2010-09-06.00:14:32
I think it doesn't make sense to display the hashes directly. That would work for
one hash maybe, but not for several, long hashes. All in all, there is a lot of
additional information about files that MirrorBrain now stores.

The current solution is to display a "Details" link, nothing else. And that links
to the page which shows the complete known metadata.

As such, this issue can be closed as resolved.

```
History
         Date         User  Action             Args
2010-09-06 00:14:32 poeml set    status: deferred -> resolved
                                   messages: + msg225
2010-03-08 21:23:01 poeml set    messages: + msg143
2010-03-08 21:20:14 poeml create

```
